modules/pricer/src/main/java/com/opengamma/strata/pricer/credit/IsdaCdsProductPricer.java

```diff
@@ -200,14 +201,14 @@ public CurrencyAmount presentValue(
    * @param refData  the reference data
    * @return the present value sensitivity
    */
-  public PointSensitivityBuilder presentValueSensitivity(
+  public PointSensitivities presentValueSensitivity(
```

Comment:
Sorry for confusion, but this would be an incompatible change, so we can't do it. If all the other methods on the class return a builder already then that is fine and the new method should too.